src/listr/tasks/setup/setupLocalPresetTaskFactory.js

```diff
@@ -41,6 +41,19 @@ function setupLocalPresetTaskFactory(
           });
         },
       },
+      {
+        title: 'Enable debug logs',
+        enabled: (ctx) => ctx.debugLogs === null,
```

Comment:
This won't work because it will be false by default. I guess you need to set `null` as default.

Reply:
Ah, I see how it works now! So it can basically have three meaningful states now: `true`, `false` and `null`?